packages/turbo-types/src/types/config.ts

```diff
@@ -105,11 +105,13 @@ export interface RootSchema extends BaseSchema {
   globalPassThroughEnv?: null | Array<EnvWildcard>;
 
   /**
+   * Deprecated as of Turborepo 2.0.0. Consider using `globalDependencies` instead.
```

Comment:
```suggestion
   * @deprecated as of Turborepo 2.0.0. Consider using {@link RootSchema.globalDependencies} instead.
```

In addition to using `@deprecated`, you can use the `@link` tag to point to the new value to use, directly